src/main/scala/syntax/GmosNorthDisperser.scala

```diff
@@ -38,8 +38,8 @@ final class GmosNorthDisperserOps(val self: GmosNorthDisperser) extends AnyVal {
   }
 
   /** Resolution at λ with the specified slit width (arcsec). */
-  def resolution(λ: Wavelength, slitWidth: Double): Int =
-    ((Wavelength.fromNanometers.reverseGet(λ) / Δλ) * (0.5 / slitWidth)).toInt
+  def resolution(λ: Wavelength, slitWidth: Angle): Int =
+    ((Wavelength.fromNanometers.reverseGet(λ) / Δλ) * (0.5 / Angle.signedArcseconds.get(slitWidth))).toInt
```

Comment:
The `AngleOptics` don't provide exactly what we need.  `Angle.arcseconds` is a `Wedge` down to an `Int` which is no good here.  `Angle.signedArcseconds` gives a `BigDecimal` which is closer but of course we don't want the "signed" part of it.  At any rate, we're talking about slit widths so nothing will be over 5 arcseconds or so which means the "signed" part is irrelevant. Still, I think ideally `Angle.arcseconds` should be renamed `Angle.arcsecondsWedge` and an `Angle.arcseconds` that produces a `BigDecimal` would be provided instead.  Does that make sense?